dask/dataframe/io/io.py

```diff
@@ -395,6 +411,14 @@ def from_bcolz(x, chunksize=None, categorize=True, index=None, lock=lock, **kwar
         return result
 
 
+@_deprecated(after_version="2022.02.1")
+def from_bcolz(*args, **kwargs):
```

Comment:
Suggesting we just remove this method in https://github.com/dask/dask/pull/9479

Reply:
Works for me :)